lib/iolog/iolog_mkdirs.c

```diff
@@ -116,11 +116,12 @@ iolog_mkdirs(const char *path)
 	    if (errno == EACCES && !uid_changed) {
 		/* Try again as the I/O log owner (for NFS). */
 		uid_changed = iolog_swapids(false);
-		if (uid_changed)
+		if (uid_changed) {
```

Comment:
The changes to this file should be dropped, the code is correct as-is.  We want to display a warning when the initial mkdirat() fails if we don't retry after swapping ids.